README.md

```diff
@@ -1 +1,107 @@
-# Blueprint
\ No newline at end of file
+# Blueprint
+
+aka **How We Collaborate** :zap:
+
+##Contents
+
+[Overview](#overview)  
+[Why Process?](#why-process)  
+[Team](#team)  
+[Workflow](#workflow)  
+[Sprints](#sprints)  
+[Notes](#notes)  
+[Contributors](#contributors)  
+
+##Overview
+
+This document is an overview of the [Vimeo Mobile and TV Group's](#notes)<sup>1</sup> product development process (a process based on the [scrum](#notes)<sup>2</sup> and [agile](#notes)<sup>3</sup> methodologies). It describes [why process](#why-process) is important and outlines three aspects of our process: [team](#team), [workflow](#workflow), and [sprints](#sprints). This document focuses on high-level guidelines rather than specific implementation tactics in order to leave room for teams to apply them as they see fit. 
+
+##Why Process?
+
+Process helps us to:
+ 
+1. Produce high quality work at a predictable [pace](#notes)<sup>4</sup>, 
+3. Promote the development of an extraordinary team.
+ 
+##Team
+ 
+A product team is a co-located, cross-functional group of roughly 4-7 people who collaborate to bring ideas to life. Proximity builds relationships. Cross-functionality fosters autonomy and self-sufficiency. Limiting team size promotes ownership.
+ 
+![Team composition](https://github.com/vimeo/Blueprint/blob/working-draft/assets/team-composition.png)    
```

Comment:
This image makes it look like Product and Dev don't interact, is that what you intended?